opendbc/car/gm/values.py

```diff
@@ -58,6 +58,37 @@ def __init__(self, CP):
     self.BRAKE_LOOKUP_BP = [self.ACCEL_MIN, max_regen_acceleration]
     self.BRAKE_LOOKUP_V = [self.MAX_BRAKE, 0.]
 
+class WMI:
+    class CHEVROLET:
+        TRUCK = {"1GC", "2GC", "3GC"}
+        MPV = {"1G1", "1GN", "2G1", "2GN", "3G1", "3GN", "KL1", "KL7"}
```

Comment:
can you add a link to some documentation for these codes? wikipedia, forum, etc.